src/components/Navigation/TopLevelBottomTabBar/index.tsx

```diff
@@ -56,15 +55,18 @@ function TopLevelBottomTabBar({state}: TopLevelBottomTabBarProps) {
     }, [shouldDisplayBottomBar]);
 
     return (
-        <View style={styles.topLevelBottomTabBar(isReadyToDisplayBottomBar, shouldUseNarrowLayout, paddingBottom)}>
-            {/* We are not rendering BottomTabBar conditionally for two reasons
+        <>
+            <View style={styles.topLevelBottomTabBar(isReadyToDisplayBottomBar, shouldUseNarrowLayout, paddingBottom)}>
+                {/* We are not rendering BottomTabBar conditionally for two reasons
                 1. It's faster to hide/show it than mount a new when needed.
                 2. We need to hide tooltips as well if they were displayed. */}
-            <BottomTabBar
-                selectedTab={selectedTab}
-                isTooltipAllowed={isReadyToDisplayBottomBar}
-            />
-        </View>
+                <BottomTabBar
+                    selectedTab={selectedTab}
+                    isTooltipAllowed={isReadyToDisplayBottomBar}
+                />
+            </View>
+            <SidePane />
+        </>
     );
 }
```

Comment:
Maybe we should create a separate RootNavigatorExtraContent component where the TopLevelBottomTabBar and the SidePane is combined instead of putting it here?

Reply:
Done 👌